cpp/ql/lib/experimental/semmle/code/cpp/semantic/SemanticBound.qll

```diff
@@ -5,6 +5,7 @@
 private import SemanticExpr
 private import SemanticExprSpecific::SemanticExprConfig as Specific
 private import SemanticSSA
+private import semmle.code.cpp.Location // TODO: SemLocation?
```

Comment:
Yeah, I don't think we should start importing `cpp` files into this otherwise language-neutral file.